### PR TITLE
London class 7 - Abdulaziz-Alhasan- Javascript-1- Week2

### PR DESCRIPTION
**Pull request by AbdulazizAlhasan**
# Your Details

- Your Name: Abdulaziz Alhasan
- Your City: London
- Your Slack Name: Abdulaziz Alhasan

# Homework Details

- Module: JS1-week2
- Week: 2

### CODE REVIEW

File: mandatory/3-playing-computer.js [hunk 1]
```diff
@@ -6,13 +6,13 @@
...
-  1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
-  3. What is printed to the console?
-  4. How many times is "f1" called?
-  5. How many times is "f2" called?
-  6. What value does the "a" parameter take in the first "f1" call?
-  7. What is the value of the "a" outer variable when "f1" is called for the first time?
+  1. This program throws an error. Why? (If you can't find it, try executing it). b is not defined 
+  2. Remove the line that throws the error. removed line 31 console.log(b);
+  3. What is printed to the console? values of x, a, d, e
```
Comment (ademalagoz) [inline mandatory/3-playing-computer.js:11]: I think it should be // 2,6,4,9,6,13,8

**Review by Hozan94 — COMMENTED** [review]
Hi @AbdulazizAlhasan , Well done on completing the exercises, you did them all greatly, I left some suggestions to consider as well.
Thanks.

File: exercises/C-comparison-operators/exercise.js [hunk 2]
```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
```
Comment (ademalagoz) [inline exercises/C-comparison-operators/exercise.js:8]: Actually we need to use variables names for all of these groups like below.

var moreStudentsThanMentors = studentCount > mentorCount; 

var enoughSpaceInRoom = roomMaxCapacity >= 25; 

var sameName = personA === personB;

**Comment by ademalagoz** [comment]
Great work Abdulaziz. I left some comments to reconsider them. Well done

**Comment by rfloresram** [comment]
You can use variables in your homework, I think the purpose of the task is to start from the easy solutions, little by little we will improve with more complex solutions! Congratulations for such a good development in your code.

**Pull request closed without merging**